canonical-websites/www.canonical.com
# Handle errors with canonicalwebteam.get-feeds

Replace local feed handling with canonicalwebteam.get-feeds - so errors are handled more effectively.

QA
--

`./run`

Then go to `/careers`, `/about`, `/partners` and check all the feeds work.

For extra marks, break some feeds by changing the feed URLs in the templates to erroneous ones.

Reload the pages and check you see an error message.
